cds/modules/previewer/templates/cds_previewer/macros/player.html

```diff
@@ -14,7 +15,9 @@
         ui : {
           width : '100%',
```

Comment:
You should remove the width and height, they are useless if `fluid:true`, and actually if `fluid:false` or missing, it will resize the player to 100px, and not 100%...